app/api/v2/managers/schedule_api_manager.py

```diff
@@ -0,0 +1,37 @@
+from marshmallow.schema import SchemaMeta
+from typing import Any
+
+from app.api.v2.managers.operation_api_manager import OperationApiManager
+from app.objects.c_operation import OperationSchema
+from app.utility.base_world import BaseWorld
+
+
+class ScheduleApiManager(OperationApiManager):
+    def __init__(self, services):
+        super().__init__(services)
+        self.services = services
+
+    def find_and_update_object(self, ram_key: str, data: dict, search: dict = None):
+        return super(OperationApiManager, self).find_and_update_object(ram_key, data, search)
```

Comment:
isn't the `data` parameter different in that it contains the operation data within the `task` key? In other words, there's an extra layer up top, right?

Reply:
Yep - the data here contains all of the fields for a `Schedule` object, including the `task`/scheduled operation. Here I was trying to call the "grandparent" class (`BaseApiManager`) because the `OperationApiManager` class has functionality specific to operations

Reply:
okay!